frontend/src/v5/ui/components/shared/modals/templates/alertModal/alertModal.component.tsx

```diff
@@ -42,9 +42,9 @@ export const AlertModal: React.FC<IAlertModal> = ({ onClickClose, currentActions
 		<Container>
 			<WarningIcon />
 			<DialogTitle>
-				<Trans
+				<FormattedMessage
 					id="alertModal.header"
-					message="Something went wrong when {currentActions}"
+					defaultMessage="Something went wrong when {currentActions}"
```

Comment:
The default messages are a duplicate of what's in messagesEn. I suppose this is in case the Intl bit is not working?

Reply:
It serves for a several purposes; extract works by extracting these defaultMessages and the id, this provides a default catalog messagesEn. This catalog is being used to see what changes from one extract run to the next one,  a diff if you will.  This lets the system not overwrite the other translation catalogs if the defaultMessage didnt changed, only updating the entries that changed or were removed/added. 
Also the english catalog is  not imported in intl, because this provides the default language (english) messages making that unnecessary. It guarantees that the version in english have all the content that is needed.  
Finally it provides a way for anyone that looks at the code to know what it should look/say without having to switch between the catalog file and the source code.

Reply:
I could rename the file messagesEn to defaultLanguage just to make it clear that is not being used as the same way as the rest of the catalog files.

Reply:
@sanmont3drepo OK, that's fine then! I didn't realise the messagesEn was being generated from this. We don't need to make any changes.